server/api/core/core.js

```diff
@@ -14,6 +14,7 @@ export default {
     // start job server
     Jobs.startJobServer(() => {
       Logger.info("JobServer started");
+      Hooks.Events.run("onJobServerStart", this);
```

Comment:
There's no object to alter for this particular hook, so no arg is needed there.  The `this` isn't really useful for anything there.  Additionally, since it's in an arrow function, the `this` is actually from the outer scope (which is just an anonymous function).  So it's certainly not breaking anything, but it's not ever going to be doing any useful either.